Source/Blazorise/Components/MessageAlert/MessageAlert.razor.cs

```diff
@@ -180,7 +181,7 @@ protected virtual bool ShowCloseButton
     /// Gets the OK button text.
     /// </summary>
     protected virtual string OkButtonText
-        => Options?.OkButtonText ?? "OK";
+        => Options?.OkButtonText ?? Localizer["OK"] ?? "OK";
```

Comment:
What's up with the capital OK, was it always like that?

Reply:
It was always a fallback, OK, But the default value was set to Ok. Google said that capitalized OK is the way to go.